assets/css/admin-custom.css

```diff
@@ -117,3 +117,54 @@ div.question_boolean_fields { margin-bottom: 10px; }
 .sensei-dismissible-link {
     text-decoration: none;
 }
+
+/**
+ * Sensei custom navigation
+ */
+.sensei-custom-navigation {
+	display: flex;
+	flex-flow: row wrap;
+	align-items: flex-start;
+	color: #000000;
+	margin-bottom: 20px;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__title h1 {
+	color: #1E1E1E;
+	font-size: 20px;
+	margin-right: 25px;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__navbar {
+	display: flex;
+	flex-flow: row wrap;
+	align-items: center;
+	justify-content: flex-start;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__navbar a {
+	margin-left: 0;
+	margin-right: 20px;
+}
+
+.sensei-custom-navigation .sensei-custom-navigation__navbar a.page-title-action {
```

Comment:
The `New course` button is moving when clicked.

![Screen Capture on 2022-01-24 at 16-14-43](https://user-images.githubusercontent.com/1612178/150799068-584db261-fd24-41d7-bf96-038a537a0a6b.gif)

Reply:
I fixed this weird bug with no efforts during the layout change. Still not sure where it came from.

Reply:
Hm, it's still present on my end. The click effect changed a bit. 🙂 

![Screen Capture on 2022-01-25 at 23-17-39](https://user-images.githubusercontent.com/1612178/151061348-5548b1b4-df50-45c6-86ca-cfc4ab87ac18.gif)

Reply:
That's weird. I couldn't reproduce it. However, I think I found the issue. It was fixed here: https://github.com/Automattic/sensei/pull/4672/commits/c6e86a7f2139c2e2754e5f5ba843b3f42dc69a2f